app/views/layouts/application.html.haml

```diff
@@ -0,0 +1,20 @@
+!!!
+%html
+  %head
+    %title InstagramChallenge
+    = stylesheet_link_tag 'application', media: 'all',
+    'data-turbolinks-track' => true
+    = javascript_include_tag 'application', 'data-turbolinks-track' => true
+    = csrf_meta_tags
+
+  %body
+    - if user_signed_in?
+      = link_to 'Sign out', destroy_user_session_path, method: :delete
+
+    = alert
+    = notice
+
+    .container{:id => 'main-container'}
```

Comment:
Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace